lita.gemspec

```diff
@@ -46,7 +46,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "rack", "~> 2.0"
   spec.add_runtime_dependency "rb-readline", "~> 0.5.0"
   spec.add_runtime_dependency "redis-namespace", "~> 1.6"
-  spec.add_runtime_dependency "thor", "~> 0.20.0"
+  spec.add_runtime_dependency "thor", "~> 1.0"
```

Comment:
Addressed in https://github.com/litaio/lita/commit/9bb5951dd9a71f81b4f2ce4f194869d13765ccc3#diff-ee0dfbe0509abc47c944702671133788R33